nvbench/detail/measure_cold.cuh

```diff
@@ -222,7 +222,10 @@ private:
 
   void run_trials()
   {
-    kernel_launch_timer timer(*this);
+    // do not use blocking kernel if benchmark is only run once, e.g., when profiling
+    // ref: https://github.com/NVIDIA/nvbench/issue/242
+    const bool disable_blocking_kernel = m_run_once || m_disable_blocking_kernel;
+    kernel_launch_timer timer(*this, disable_blocking_kernel);
```

Comment:
```suggestion
    kernel_launch_timer timer(*this, m_disable_blocking_kernel);
```

Reply:
@alliepiper So are you suggesting `nvbench::state::set_run_once` and `nvbench::benchmark_base::set_run_once` to also set `m_disable_blocking_kernel` to `true` if `m_run_once` flag is to be set to `true`?